ui/page/transaction/transactions_page.go

```diff
@@ -54,27 +51,45 @@ type TransactionsPage struct {
 
 	selectedTabIndex int
 
-	txTypeDropDown   *cryptomaterial.DropDown
+	txTypeDropDown *cryptomaterial.DropDown
+	walletDropDown *cryptomaterial.DropDown
+
 	transactionList  *cryptomaterial.ClickableList
 	previousTxFilter int32
 	scroll           *components.Scroll
 
-	tabs *cryptomaterial.ClickableList
+	tab *cryptomaterial.SegmentedControl
 
 	materialLoader material.LoaderStyle
+
+	sourceWalletSelector *components.WalletAndAccountSelector
+	assetWallets         []sharedW.Asset
+	selectedWallet       sharedW.Asset
+
+	isHomepageLayout,
+	showLoader,
+	showDisabledLayout bool
 }
 
-func NewTransactionsPage(l *load.Load) *TransactionsPage {
+func NewTransactionsPage(l *load.Load, isHomepageLayout bool) *TransactionsPage {
 	pg := &TransactionsPage{
 		Load:             l,
 		GenericPageModal: app.NewGenericPageModal(TransactionsPageID),
 		separator:        l.Theme.Separator(),
 		transactionList:  l.Theme.NewClickableList(layout.Vertical),
+		isHomepageLayout: isHomepageLayout,
+	}
+
+	pg.tab = l.Theme.SegmentedControl(txTabs)
```

Comment:
This segmented control is only necessary if there is at least 1 dcr wallet in the app. Otherwise, don't show this.